frontend/src/app/projects/models/create-coworking-method/create-coworking-method.component.html

```diff
@@ -223,9 +223,15 @@ <h2>
             Back
           </a>
 
-          <button mat-raised-button color="primary" (click)="enableEditing()">
-            Edit
-          </button>
+          <div>
+            <button mat-raised-button color="warn" (click)="deleteGitModel()">
+              Delete
+            </button>
```

Comment:
Here I would change the name of the button to `Unlink` (or a similar term), since we are not actually deleting the git model.